src/sst/core/baseComponent.cc

```diff
@@ -312,13 +319,15 @@ BaseComponent::configureLink(const std::string& name, TimeConverter* time_base,
                     my_info->link_map = myLinks;
                 }
                 myLinks->insertLink(name,tmp);
+#ifndef SST_ENABLE_PREVIEW_BUILD
                 // Need to set the link's defaultTimeBase to nullptr,
                 // except in the case of this being an Anonymously
                 // loadeed SubComponent, then for backward
                 // compatibility, we leave it as is.
                 if ( !my_info->isLegacySubComponent() ) {
                     tmp->setDefaultTimeBase(nullptr);
```

Comment:
Should defaulttimebase be set to nullptr for all subcomponents if SST_ENABLE_PREVIEW_BUILD is enabled? So the "if" should be ifdef'd but not the call in it? Or is there code elsewhere that takes care of it?

Reply:
Fixed and pushed.